integrationTests/DonationPersistenceTest.php

```diff
@@ -39,7 +40,7 @@ public function testItLoadsADonationToDB(): void
         $donation = $donationRepo->findOneBy(['uuid' => $donationRow['uuid']]);
 
         $this->assertNotNull($donation);
-        $this->assertSame(DonationStatus::Refunded, $donation->getDonationStatus());
+        $this->assertEquals(DonationStatus::Refunded, $donation->getDonationStatus());
```

Comment:
just changed this because PHPUnit gives a much more useful failure message with assertEquals on an enum than assertSame. If assertSame failed it just says two objects are not identical and doesn't elaborate. if assertEquals fails we actually get to see what the expected and actual enum are.